### PR TITLE
Refactor TS debris to work more like in the original

### PR DESCRIPTION
**Pull request by reaperrr**
In TS, debris weren't cosmetic and explosion-less like in RA2, they exploded on impact and dealt damage.
Also, some of them had trails.

This PR tries to approximate the original flight characteristics, damage and visuals as closely as possible.

Remaining differences to original game:
- small debris did not play an explosion effect and sound in the original, but that was due to a typo by Westwood. I enabled the explosion animation (`twlt026`), but not the sound, since `expnew05.aud` is quite annoying.
- all debris additionally displayed and alpha glow on impact.

### CODE REVIEW

**Review by pchote — APPROVED** [review]

File: mods/ts/weapons/otherweapons.yaml [hunk 1]
```diff
@@ -106,3 +106,62 @@ Veins:
...
+	Inherits: ^Debris
+	Projectile: Bullet
+		Image: dbrislg
+		Sequences: 2, 3, 4, 6, 7, 9, 10
+	Warhead@1Dam: SpreadDamage
+		Spread: 320
+		Damage: 20
+	Warhead@2Eff: CreateEffect
+		Explosions: small_twlt
+		ExplosionPalette: effectalpha75
```
Comment (abcdefg30) [inline mods/ts/weapons/otherweapons.yaml]: Isn't this already defined on `^Debris`?

File: mods/ts/weapons/otherweapons.yaml [hunk 2]
```diff
@@ -106,3 +106,62 @@ Veins:
...
+		ImpactSounds: expnew06.aud
+		InvalidImpactTypes: Water
```
Comment (abcdefg30) [inline mods/ts/weapons/otherweapons.yaml]: Same question here.

**Review by abcdefg30 — APPROVED** [review]
Looks good otherwise.

**Comment by reaperrr** [comment]
Updated.

**Pull request merged**